google/cloud/bigquery_v2/types/model.py

```diff
@@ -103,6 +103,7 @@ class Model(proto.Message):
 
     class ModelType(proto.Enum):
         r"""Indicates the type of the Model."""
+
```

Comment:
these formatting changes are benign, but it's not clear why they're in this PR.  Maybe pull out unrelated cleanups into a separate PR?

Reply:
Indeed, I will revert these changes